validator/project_validator.go

```diff
@@ -421,6 +421,18 @@ func validateProjectConfigPeriodicBuilds(pc *model.ProjectConfig) ValidationErro
 
 func validateProjectConfigAliases(pc *model.ProjectConfig) ValidationErrors {
 	errs := []string{}
+	for i := range pc.GitTagAliases {
+		pc.GitTagAliases[i].Alias = evergreen.GitTagAlias
+	}
+	for i := range pc.GitHubChecksAliases {
+		pc.GitHubChecksAliases[i].Alias = evergreen.GithubChecksAlias
+	}
+	for i := range pc.CommitQueueAliases {
+		pc.CommitQueueAliases[i].Alias = evergreen.CommitQueueAlias
+	}
+	for i := range pc.GitHubPRAliases {
+		pc.GitHubPRAliases[i].Alias = evergreen.GithubPRAlias
+	}
```

Comment:
Can you please explain what this does? Why was it not needed before, but needed now?

Reply:
Can you please also expand on the [explanation here](https://github.com/evergreen-ci/evergreen/blob/main/model/project_aliases.go#L59) so that it's easier for future engineers to grok?

Reply:
Alias validation expectes the `Alias` field to be [populated](https://github.com/evergreen-ci/evergreen/blob/main/model/project_aliases.go#L495). When defining aliases in YAML it shouldn't be necessary to define stuff like `alias: _github` manually since we can instead infer the alias name depending on what alias list it is in. I.e. for all `GitHubPRAliases` we'll add the relevant Github PR alias to it.

Reply:
Added some more documentation